### ISSUE TITLE
Error in new theming system

### ISSUE THREAD

**Issue by MBach** [callstack/react-native-paper#3528]
### Current behaviour
Trying to use new theming system with `react-navigation`:

```
const { LightTheme } = adaptNavigationTheme({ reactNaivgationLight: DefaultTheme })
```

Makes my app crash at startup.
Two things here:

- TypeError: Cannot read property 'colors' of undefined
- Typography error with `reactNaivgationLight` (should be `reactNavigationLight`)

### Expected behaviour
Well... To work.

### How to reproduce?
Just copy/paste the line in your code

### Environment

| software                      | version
| --------------------- | -------
| react-native                | 0.70.6
| react-native-paper    | 5.0.1

**Comment by lukewalczak** [comment]
Hey @MBach, the typo was fixed and released in `5.0.2`. Does the `DefaultTheme` passed to the function come from react-navigation? Additionally please provide the snack presenting the issue.

**Comment by lukewalczak** [comment]
@MBach has your problem been solved?
